maintainers/maintainer-list.nix

```diff
@@ -85,6 +85,12 @@
     githubId = 56617252;
     matrix = "@oxc45:matrix.org";
   };
+  _0xd61 = {
```

Comment:
Put this in a separate commit: `maintainers: add 0xd61`

Reply:
done